scripts/operations/configuration/re_run_node_personalization.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/bash
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+################################################################################
+#
+# The script will clear the CFS state for the selected nodes and re-run the
+# configuration
+#
+# Usage: re_run_node_personalization.sh
+#
+################################################################################
+locOfScript=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+# Inform ShellCheck about the file we are sourcing
+# shellcheck source=./bash_lib/common.sh
+. "${locOfScript}/bash_lib/common.sh"
+
+DIFF_SCRIPT="${locOfScript}/json_str_list_diff.py"
+
+get_xnames_list() {
+  cray hsm state components list --role Management --type Node --format json | jq -r '[.Components[].ID]'
+}
+
+get_rpm_version() {
+  rpm -q --qf "%{VERSION}" "$1"
+}
+
+xnames_list=$(get_xnames_list)
+xnames=$(echo "$xnames_list" | jq -r 'join(",")')
+xnames_arr=$(echo "$xnames_list" | jq -r '.[]')
+craycli_version=$(get_rpm_version "craycli")
+
+echo "Cray CLI version: ${craycli_version}"
+echo "Clearing CFS state for ${xnames}"
+
+if [ "$(printf '%s\n' "$craycli_version" "0.90.0" | sort -V | head -n1)" = "$craycli_version" ]; then
+  FAILED=""
+  COUNT=0
+  for xname in "${xnames_arr[@]}"; do
+    echo "Clearing CFS state of ${xname}"
+    cray cfs v3 components update --error-count 0 --state '[]' --format json "${xname}" && let COUNT+=1 || FAILED+=" ${xname}" \
+      || err_exit "Command failed: cray cfs v3 components update --error-count 0 --state '[]' --format json '${xname}' && let COUNT+=1 || FAILED+='${xname}'"
```

Comment:
This branch handles the failures differently and should not call err_exit immediately. That is what the `FAILED` and `COUNT` variables are accomplishing.
```suggestion
    cray cfs v3 components update --error-count 0 --state '[]' --format json "${xname}" && let COUNT+=1 || FAILED+=" ${xname}"
```